### PR TITLE
Constaint python and added config for renovate bot

### PR DESCRIPTION
**Pull request by ckim328**
### Background 
Further fine tunes our renovate bot config setup

### Fixes 
Applying some of the results of the survey. This is an example PR, but it adds stability days, concurrent PR limits, and adds config to bypass that if there is a security vulnerability.
### Change Summary
* Sets prConcurrentLimit to 5, stabilityDays to 7
* Sets both prConcurrentLimit and stabilityDays to 0 if it is a CVE is reported
* Also constrains python to 3.7 (before, even though it said `"python":"3.7"`, it wasn't working. Followed up with the maintainer on why that was happening, but the solution was to use `"python":"~=3.7.0"` instead
* Have Renovate Bot add labels (e.g., `lang: java` and `dependencies`).
### Additional Notes
Note: we also have dependabot (which is built into Github) to notify us of security vulnerabilities. 

### Testing Procedure
The renovate bot PRs will decrease

### Related PRs or Issues 
n/a

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -10,9 +10,27 @@
...
+    "addLabels": ["lang: python"]
+  },
+  "java":{
+    "addLabels": ["lang: java"]
+  },
+  "golang":{
+    "addLabels": ["lang: go"]
+  },
+  "nuget":{
+    "addLabels": ["lang: C#"]
```
Comment (NimJay) [inline renovate.json:34]: Suggestion: I looked at the existing labels. `lang: C#` is not one of them for some reason. 🤷  Maybe we'll just use `lang: dotnet`.

**Review by NimJay — CHANGES_REQUESTED** [review]
Thanks so much, @ckim328, for your continual effort on this Renovate Bot stuff!
Left one comment.

File: renovate.json (hunk 1 above)
Comment (ckim328) [reply renovate.json:34]: that sounds good to me!

**Review by charlieyu1996 — APPROVED** [review]

**Review by NimJay — APPROVED** [review]
Looks good to me!
Thanks for this change, Christine.

**Pull request merged**